app/views/questionnaire.py

```diff
@@ -170,13 +176,15 @@ def post_household_composition(routing_path, **kwargs):
     return _render_page(routing_path, block, current_location, post_form=form)
 
 
-@questionnaire_blueprint.route('thank-you', methods=['GET'])
-def get_thank_you(eq_id, form_type, collection_id):  # pylint: disable=unused-argument
-    survey_completed_metadata = current_app.eq['session_storage'].get_survey_completed_metadata()
-    schema = load_schema_from_params(eq_id, form_type)
+@after_submission_blueprint.route('/thank-you', methods=['GET'])
+def get_thank_you():  # pylint: disable=unused-argument
```

Comment:
you can remove the pylint exclusion as there are no unused arguments on this funciton